test/modules/number.spec.ts

```diff
@@ -625,6 +635,56 @@ describe('number', () => {
         );
       });
     });
+
+    describe('romanNumeral', () => {
+      it('should generate a Roman numeral within default range', () => {
+        const roman = faker.number.romanNumeral();
+        expect(roman).toBeTypeOf('string');
+        expect(roman).toMatch(/^[IVXLCDM]+$/);
+      });
+
+      it('should generate a Roman numeral with max value of 1000', () => {
+        const roman = faker.number.romanNumeral(1000);
+        expect(roman).toMatch(/^[IVXLCDM]+$/);
+      });
+
+      it('should generate a Roman numeral between 1 and 10', () => {
+        const values = [
+          'I',
+          'II',
+          'III',
+          'IV',
+          'V',
+          'VI',
+          'VII',
+          'VIII',
+          'IX',
+          'X',
+        ];
+
+        const roman = faker.number.romanNumeral({ min: 1, max: 10 });
+
+        expect(values).toContain(roman);
+      });
```

Comment:
Sorry, I didn't think of this earlier.
I changed this test to check for the actual values:

```suggestion
      it.each(
        Object.entries({
          I: 1,
          IV: 4,
          IX: 9,
          X: 10,
          XC: 90,
          XCIX: 99,
          XXVII: 27,
          CCLXIII: 263,
          DXXXVI: 536,
          DCCXIX: 719,
          MDCCCLI: 1851,
          MDCCCXCII: 1892,
          MMCLXXXIII: 2183,
          MMCMXLIII: 2943,
          MMMDCCLXVI: 3766,
          MMMDCCLXXIV: 3774,
          MMMCMXCIX: 3999,
        })
      )(
        'should generate a Roman numeral %s for value %d',
        (expected: string, value: number) => {
          const mock = vi.spyOn(faker.number, 'int');
          mock.mockReturnValue(value);

          const actual = faker.number.romanNumeral();

          mock.mockRestore();

          expect(actual).toBe(expected);
        }
      );
```

A few manually selected ones and a few random ones.

Reply:
Done!!